core/src/main/python/update.py

```diff
@@ -44,12 +46,12 @@
 
 __required_arguments = [
     CommandLineArgUtil.ORACLE_HOME_SWITCH,
-    CommandLineArgUtil.DOMAIN_HOME_SWITCH,
```

Comment:
Same questions as for Deploy.  For discoverDomain -remote, we still require the user to provide -domain_home <remote-domain-home-path> to support tokenization.  I would think we might need this for at least updateDomain -remote if not all remote operations.